### ISSUE TITLE
Could you provide evaluation codes for NExT-QA, STAR and TVQA on video_chat2?

### ISSUE THREAD

**Issue by CUCldyyyyy** [OpenGVLab/Ask-Anything#89]
Hey,your work is really impressive! Could you provide evaluation codes for NExT-QA, STAR and TVQA ,it seems some changes must be made on original mvbench.ipynb file. I'd be appreciate if you could.Thanks again!

**Comment by Andy1621** [comment]
For NExT-QA, STAR and TVQA, we simply change the code as in [mvbench.ipynb](https://github.com/OpenGVLab/Ask-Anything/blob/main/video_chat2/mvbench.ipynb). What you need to do is preparing the corresponding dataset use using the same testing prompt.

**Comment by Andy1621** [comment]
You can follow [SeViLA](https://github.com/Yui010206/SeViLA/tree/main/sevila_data) to prepare the dataset, and change the code to load the JSON.

**Comment by CUCldyyyyy** [comment]
I see，thanks for your reply.

**Comment by CUCldyyyyy** [comment]
hey!A problem occured to my inference code,could u tell me the possible reason?

`Question: why does the owl fly back to the man in green and land on the arm of the lady in white?
Options:
(A) defend itself.
(B) green man instructed owl.
(C) greeting lady.
(D) escape man.
(E) find food.
Only give the best option.
###Assistant: Best option:(
(bolds君 — †</s>
GT: (B) green man instructed owl.
Part  Acc: 0.00%
Total Acc: 0.00%`

**Comment by CUCldyyyyy** [comment]
messy code is given in Best option,error is :IndexError: piece id is out of range.

**Comment by Andy1621** [comment]
You might use the wrong version of Vicuna-v0, please check https://github.com/OpenGVLab/Ask-Anything/issues/81

**Comment by CUCldyyyyy** [comment]
Thank you for your response! I have resolved the issue. Additionally, I noticed that the suffixes in the fine-tuning weights for the three released stages include '7b'. When I directly use 'vicuna-13b-v0', I encounter a dimension mismatch error during the model weight loading (4096 vs. 5120). How can this be resolved? Do I need to modify the source code dimensions, or is it due to the absence of a 13b version in the currently released fine-tuning weights?

**Comment by Andy1621** [comment]
Yes! The model needs to be retrained with a new LLM. Currently, we do not release the 13B model for its marginal improvement~

**Comment by CUCldyyyyy** [comment]
Hey！I need to run inference on MSVD-QA,wondering how to change the original mvbench.ipynb file to fit the open-ended dataset? It seems designed for multi-choice task and the prompt need to be revised for op task. Could u provide the method to reproduce the result on MSVD-QA?Thanks a lot!

**Comment by Andy1621** [comment]
Please check the code in [Video_ChatGPT](https://github.com/mbzuai-oryx/Video-ChatGPT/tree/main/quantitative_evaluation).
